File: HPWorkpath_v1.6.3/Samples/ExampleExtensions/source/GoogleSigninSample/src/com/hp/workpath/sample/googlesigninsample/task/TaskInterface.java
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.googlesigninsample.task;

import com.hp.workpath.sample.googlesigninsample.model.AccountInfo;

public interface TaskInterface {
    void onFailure(Throwable t);

    void refreshedToken(String token);

    void revokedToken();

    void receivedToken(AccountInfo accountInfo);
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleExtensions/source/GoogleSigninSample/src/com/hp/workpath/sample/googlesigninsample/task/TaskInterface.kt
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.googlesigninsample.task

import com.hp.workpath.sample.googlesigninsample.model.AccountInfo

interface TaskInterface {
    fun onFailure(t: Throwable?)
    fun refreshedToken(token: String)
    fun revokedToken()
    fun receivedToken(accountInfo: AccountInfo)
}